source/tools/glkernel-cli/JsonImporter.cpp

```diff
@@ -0,0 +1,124 @@
+#include "JsonImporter.h"
+
+#include "helper.h"
+
+#include <glkernel/Kernel.h>
+
+#include <cppassist/logging/logging.h>
+
+void forEachCell(cppexpose::VariantArray * depthArray, std::function<void(const cppexpose::Variant&)> lambda)
```

Comment:
I suggest you provide an `forEachCell` overload where the lambda takes the current index as second parameter.